Sources/Testing/Traits/TimeLimitTrait.swift

```diff
@@ -39,18 +63,123 @@ extension Trait where Self == TimeLimitTrait {
   ///
   /// - Returns: An instance of ``TimeLimitTrait``.
   ///
+  /// Test timeouts do not support high-precision, arbitrarily short durations,
+  /// due to variability in testing environments. The time limit duration must
+  /// be at least one minute, and can only be expressed in minute-length
+  /// increments.
+  ///
   /// When this trait is associated with a test, that test must complete within
   /// a time limit of, at most, `timeLimit`. If the test runs longer, an issue
   /// of kind ``Issue/Kind/timeLimitExceeded(timeLimitComponents:)`` is
   /// recorded. This timeout is treated as a test failure.
   ///
+  /// The time limit amount specified by `timeLimit` may be reduced if the
```

Comment:
This isn't easily configurable, so we might want to explain _how_ to do that somewhere.

Reply:
Also, if the developer lowers the time limit granularity, we need to then offer a way to actually _specify_ a lower time limit.

Reply:
Ok I've reworded this to better clarify how the maximum might be controlled